src/ansys/geometry/core/geometry/surfaces/cone.py

```diff
@@ -235,37 +240,35 @@ def project_point(self, point: Point3D) -> "ConeEvaluation":
 
         return ConeEvaluation(self, ParamUV(u, v))
 
-    def get_u_parameterization(self) -> Parameterization:
+    def parameterization(self) -> tuple[Parameterization, Parameterization]:
```

Comment:
Do not use the "tuple" type yet. We need to handle that properly throughout the entire library. Use ``from beartype.typing import Tuple``

Reply:
Applies everywhere